src/amo/components/LandingPage/index.js

```diff
@@ -214,20 +233,23 @@ export class LandingPageBase extends React.Component {
           footerText={html.featuredFooterText}
           footerLink={html.featuredFooterLink}
           header={html.featuredHeader}
+          loading={loading}
```

Comment:
Sure, that could be done as an optimization in the future but both the reducer and action creator would need refactoring because right now they 1) retrieve data from the 3 API calls at once and 2) do not store a distinct loading flag for each API result. It will also make the component more complex since it will have to juggle multiple loading flags. 

The important part is that the API requests are made in parallel so at least we have that so far.